src/tests/system/tests/test_access_control_simple.py

```diff
@@ -0,0 +1,163 @@
+        3. Start SSSD
+    :steps:
+        1. Try to login as ‘user1’
+        2. Try to login as ‘user2’
+    :expectedresults:
+        1. User1 cannot login
+        2. User2 can login
+    :customerscenario: False
+    """
+
+    for i in ["user1", "user2"]:
+        provider.user(i).add()
+
+    client.sssd.domain["access_provider"] = "simple"
+    client.sssd.domain["simple_deny_users"] = "user1"
+
+    client.sssd.start()
+
+    assert not client.auth.ssh.password("user1", "Secret123"), "User should be able to log in!"
+    assert client.auth.ssh.password("user2", "Secret123"), "User should NOT be able to log in!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_access_control__simple_filter_permits_user_login_based_on_group(client: Client, provider: GenericProvider):
+    """
+    :title: Simple access filter permits user login
+    :setup:
+        1. Create users ‘user1’, ‘user2’, ‘user3’
+        2. Create group ‘group1’ with members ‘user1, user3’
+        3. Create group ‘group2’ with member ‘user2, user3’
+        4. Configure SSSD with ‘access_provider = simple’,
+        5. Configure SSSD with ‘simple_allow_groups = group1’ and ‘simple_deny_groups = group2’
+        6. Start SSSD
+    :steps:
+        1. Try to login with ‘user1’
+        2. Try to login with ‘user2’
+        3. Try to login with ‘user3’
+    :expectedresults:
+        1. User1 can login
+        2. User2 cannot login
+        3. User3 cannot login
+    :customerscenario: False
+    """
+
+    # Create users
+    for i in ["user1", "user2", "user3"]:
+        provider.user(i).add()
+
+    group1 = provider.group("group1").add()
+    group2 = provider.group("group2").add()
+
+    group1.add_members(["user1", "user3"])
+    group2.add_members(["user2", "user3"])
+
+    client.sssd.domain["access_provider"] = "simple"
+    client.sssd.domain["simple_allow_groups"] = "group1"
+    client.sssd.domain["simple_deny_groups"] = "group2"
+
+    client.sssd.start()
+
+    assert client.auth.ssh.password("user1", "Secret123"), "User should be able to log in!"
+    assert not client.auth.ssh.password("user2", "Secret123"), "User should NOT be able to log in!"
+    assert not client.auth.ssh.password("user3", "Secret123"), "User should NOT be able to log in!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_access_control__validate_behavior_of_deny_superceding_allow(client: Client, provider: GenericProvider):
+    """
+    :title: Validate if superceding deny works for individual users and groups.
+                for individual users and groups.
+    :setup:
+        1. Create users `user1` and `user2`
+        2. Create groups `allowed_group` and `denied_group` both with members `user2`
+        3. Configure SSSD with:
+           - `access_provider = simple`
+           - `simple_allow_users = user1`
```

Comment:
simple_allow_users and simple_deny_users shouldn't configured because it's a configuration if both are used. This scenario should just test simple_allow_groups and simple_deny_groups. We can simplify the text to 

```
3. Configure SSSD with 'access_provider = simple', 'simple_allow_groups = allowed_group'
    and 'simple_deny_groups = denied group'
```

This will be the first test with ```-``` , if you want to indent it, use four spaces to make it consistent. 

```
3. Configure SSSD with
    'access_provider = simple',
    'simple_allow_groups = allowed_group'
    'simple_deny_groups = denied group'
```

Reply:
Identation fixed.

My idea with this test was to:
 - If user1 is in allow_user and deny_user, the deny should supersede the allow
 - If user2 is in allow_group and deny_group, the deny should supersede the allow

We are running it once to test both options. You are telling me it shouldn't happen?